bundle/edu.gemini.pit/src/main/scala/edu/gemini/pit/ui/view/tac/TacView.scala

```diff
@@ -36,7 +36,7 @@ object TacView {
       // no partners and the tab's controls will be disabled. Elsewhere we don't bother with the
       // `gt.sub.specialType` check.
       case gt:SpecialProposalClass if gt.sub.specialType === SpecialProposalType.GUARANTEED_TIME =>
-        List(GuaranteedTimePartner -> gt.sub.response)
+        gt.sub.response.map(r => List(GuaranteedTimePartner -> Some(r))).getOrElse(Nil)
```

Comment:
`gt.sub.response.toList.tupleLeft(GuaranteedTimePartner)` might work too

Reply:
Never mind, I guess the response has to be wrapped in an option inside the list.